pitchr/music.py

```diff
@@ -802,6 +804,22 @@ def articulation(self, articulation):
         """
         self._articulation = articulation
 
+    @property
+    def tie(self):
+        """Get the tie status of a Note
+
+        :returns: tie
+        """
+        return self._tie
+
+    @articulation.setter
+    def articulation(self, tie):
```

Comment:
I think you meant to call this `tie` instead of `articulation` lol

Reply:
Got it, updated and pushed new commit to fix typos.